assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -785,11 +783,16 @@ const baseSelectors = {
 	 * @param {string} slug  Module slug.
 	 * @return {(null|Object)} Activation error for a module slug; `null` if there is no error or an error object if we cannot activate a given module.
 	 */
-	getCheckRequirementsError( state, slug ) {
+	getCheckRequirementsError: createRegistrySelector( ( select ) => ( state, slug ) => {
 		invariant( slug, 'slug is required.' );
-		const requirementsStatus = state.checkRequirementsResults[ slug ];
-		return requirementsStatus === true ? null : requirementsStatus;
-	},
+
+		// Need to use registry selector here to ensure resolver is invoked.
+		if ( select( STORE_NAME ).canActivateModule( slug ) ) {
```

Comment:
This change isn't explicitly needed by this PR but a bug I noticed during implementation.

`checkRequirements` functions were being invoked twice due to separate resolvers being used for `canActivateModule` and `getCheckRequirementsError`. Even though they use the same underlying function, the datastore will run them if present.